src/scripts.js

```diff
@@ -3,6 +3,8 @@
 import './css/base.scss';
 import { fetchTravelers, fetchSingleTraveler, fetchTrips, 
   fetchDestinations } from './apiCalls';
+import Traveler from './traveler';
```

Comment:
Great job exporting your fetch in a different file and importing it to your script!